modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipServiceImpl.java

```diff
@@ -196,6 +190,33 @@ public ObjectRelationship updateObjectRelationship(
 			deletionType, edge, labelMap, objectField);
 	}
 
+	private void _checkUpdatePermission(
```

Comment:
I think we could rename to be more generic like: `checkModelResourcePermission` and you could pass the `string actionId`